app/controllers/organizations_controller.rb

```diff
@@ -43,4 +41,23 @@ def events
       PaginationPresenter
     )
   end
+
+  def evidence_items
+    evidence_items = EvidenceItem.order('evidence_items.id DESC')
+      .eager_load(:submitter)
+      .where('users.id': user_ids)
```

Comment:
It returns the same number of results so I think it works either way? I changed the clause, just to be sure.